Tests/Engine/CustomizedRule.tests.ps1

```diff
@@ -97,19 +97,16 @@ Describe "Test importing correct customized rules" {
 		    $customizedRulePath.Count | Should Be 1
 		}
 
-        It "will show the custom rule when given a rule folder path with trailing backslash" {
+        It "will show the custom rule when given a rule folder path with trailing backslash" -skip:(!$IsWindows){
 			# needs fixing for linux
-			if (-not (Test-PSEditionCoreCLRLinux))
-			{
-				$customizedRulePath = Get-ScriptAnalyzerRule  -CustomizedRulePath $directory/samplerule/ | Where-Object {$_.RuleName -eq $measure}
-				$customizedRulePath.Count | Should Be 1
-			}
+            $customizedRulePath = Get-ScriptAnalyzerRule  -CustomizedRulePath $directory/samplerule/ | Where-Object {$_.RuleName -eq $measure}
+            $customizedRulePath.Count | Should Be 1
 		}
 
 		It "will show the custom rules when given a glob" {
 			# needs fixing for Linux
 			$expectedNumRules = 4
-			if (Test-PSEditionCoreCLRLinux)
+			if ( ! $IsWindows )
```

Comment:
Very minor but here and in other changes the spacing inside the parenthesis is inconsistent.

Reply:
sure - will fix